server/config/config.go

```diff
@@ -25,12 +25,12 @@ const (
 
 // MysqlConfig defines configs related to MySQL
 type MysqlConfig struct {
-	Protocol        string
-	Address         string
-	Username        string
-	Password        string
+	Protocol        string `yaml:"protocol"`
```

Comment:
Is this backwards compatible? I thought that without defining it it would default to `Protocol` instead of `protocol`.

Reply:
Testing and confirmed that all keys are case insensitive. Also confirmed in the docs for viper https://github.com/spf13/viper#does-viper-support-case-sensitive-keys. It seems like we added yaml tags for snake casing, leaving all single word fields alone. Should I remove the yaml tags?